ax/service/tests/test_ax_client.py

```diff
@@ -2562,61 +2562,6 @@ def get_solution(use_y0_threshold: bool, use_y2_constraint: bool) -> Set[int]:
                 return {0, 1}
             return {0, 2}
 
-            ax_client = get_client_with_simple_discrete_moo_problem(
```

Comment:
If you wonder why I removed this, this is unreachable, is this meant this way?

Reply:
Wow, good catch! I'm pretty sure this code is supposed to be run. It was accidentally indented too far during a refactoring. Could you unindent it?